docs/reference/job-status.rst

```diff
@@ -0,0 +1,84 @@
+.. _job_status:
+
+Checkbox Job Status
+===================
+
+A Checkbox job always receives a finishing status either automatically or
+manually. The following is a description of what each status means and when/how
+it is received.
+
+Passing Jobs
```

Comment:
```suggestion
Passed Jobs
```
It might be clearer to use the same term as in the output. It's also more consistent for searching